### PR TITLE
Closing Connection Section

### PR DESCRIPTION
**Pull request by Spomky**
This PR adds a section to indicate that the non-necessary connections should be closed

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -29,6 +29,19 @@ eventSource.onmessage = e => console.log(e); // do something with the payload
 
 The `EventSource` class is available [in all modern web browsers](https://caniuse.com/#feat=eventsource). And for legacy browsers, [there are polyfills](ecosystem/awesome.md#useful-related-libraries).
 
+## Closing Connection
+
+It is important to close this connection between the client and the hub if it is no longer needed.
+Opened connections have continous buffer that will be drain your application resources.
```
Comment (jdreesen) [inline docs/getting-started.md]:
```suggestion
Opened connections have a continuous buffer that will drain your application resources. 
```

File: docs/getting-started.md [hunk 2]
```diff
@@ -29,6 +29,19 @@ eventSource.onmessage = e => console.log(e); // do something with the payload
...
+This is especially true when using Single Page Applications based on ReactJS for example: the connection is maintained when the component that created it is unmounted.
```
Comment (jdreesen) [inline docs/getting-started.md]:
```suggestion
This is especially true when using Single Page Applications (based on e.g. ReactJS): the connection is maintained even if the component that created it is unmounted.
```

**Comment by Spomky** [comment]
Thank you @jdreesen. As always your suggestions are relevant!
:+1:

**Comment by dunglas** [comment]
Thanks @Spomky

**Pull request merged**